lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
the original change was straight from node-pre-gyp's documentation
https://github.com/mapbox/node-pre-gyp#3-dynamically-require-your-node

Its the same thing that sqlite3 does:
https://github.com/mapbox/node-sqlite3/blob/master/package.json#L12
https://github.com/mapbox/node-sqlite3/blob/master/lib/sqlite3.js#L1-L5